lib/workflow/stores/mongo.js

```diff
@@ -883,5 +883,23 @@ function mongoStoreFactory(waterline, Promise, Constants, Errors, assert, _) {
         return waterline.graphobjects.findOne(query);
     };
 
+    /**
+     * Finds the child graph of a runGraphJob by the job's task id/the child
+     * graph's parentTaskId
+     * @param {String} parentTaskId - The task id of a runGraphTask to match
+     * against a graph's context._parent.taskId field
+     * @returns {Promise} a promise for the graph with the given context._parent.taskId
+     * @memberOf store
+     */
+    exports.findChildGraph = function(parentTaskId) {
+        assert.uuid(parentTaskId, "Parent task ID should be a UUID");
+
+        var query = {
+            "context._parent.taskId": parentTaskId
```

Comment:
Probably just adding the attributes to the graph object model would be best and simplest